source/admin_manual/list_of_events.rst

```diff
@@ -2315,6 +2315,29 @@ indexes in metacache.
 fs-dictmap
 ----------
 
+
+fs_dictmap_dict_write_uncertain
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 2.3.13
+
+The event is sent whenever a dict write is uncertain.
```

Comment:
Could explain a bit more, like: For example writes to Cassandra may eventually succeed even if the write initially appeared to fail.